ci/test/00_setup_env_native_fuzz_with_valgrind.sh

```diff
@@ -16,3 +16,4 @@ export RUN_FUZZ_TESTS=true
 export FUZZ_TESTS_CONFIG="--valgrind"
 export GOAL="install"
 export BITCOIN_CONFIG="--enable-fuzz --with-sanitizers=fuzzer CC=clang CXX=clang++"
+export CCACHE_SIZE=200M
```

Comment:
Effecitve size is 93 MB:

https://travis-ci.org/github/bitcoin/bitcoin/jobs/708074353#L2875

How is it possible that this gives a speedup, when the default size is 100 MB?

Reply:
Still 95 after two runs: https://travis-ci.org/github/bitcoin/bitcoin/jobs/712518050#L2879